pykern/util.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""utils for pykern
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern import pkconfig
+from pykern import pkinspect
+from pykern import pkio
+from pykern.pkdebug import pkdc, pkdexc, pkdlog, pkdp
+import os.path
+import sys
+
+
+_DEFAULT_ROOT = "run"
+
+
+def cfg_db_dir_parser(value):
+    """Config value or root package's parent or cwd with `_DEFAULT_ROOT`"""
+    if not os.path.isabs(value):
+        pkconfig.raise_error("must be absolute")
+    if not os.path.isdir(value):
+        pkconfig.raise_error("must be a directory and exist")
+    return pkio.py_path(value)
+
+
+def dev_run_dir(package_object):
+    """Initialize run dir
```

Comment:
Comment like this is not useful. "Initialize" is incorrect. Describe what it does.